torch_geometric/loader/link_neighbor_loader.py

```diff
@@ -291,6 +342,11 @@ def __init__(
         if edge_label is None:
             edge_label = torch.zeros(edge_label_index.size(1),
                                      device=edge_label_index.device)
+        self.edge_label = edge_label
```

Comment:
Do we need this?

Reply:
its needed for pytorch_lightning <1.6, will add a comment.